addons/knobs/README.md

```diff
@@ -50,18 +53,33 @@ stories.add('with a button', () => (
   <button disabled={boolean('Disabled', false)} >
     {text('Label', 'Hello Button')}
   </button>
-))
+));
 
+const options = {};
 // Knobs as dynamic variables.
-stories.add('as dynamic variables', () => {
+stories.add('as dynamic variables', withKnobsOptions(options)(() => {
```

Comment:
Can we make this `withKnobs(options)`? To make it consistent with `withNotes` / `withInfo`?
OR can we make this `wrapperKnobs(options)` if we are not ready to migrate the addon API?